src/Fixture/Package.php

```diff
@@ -195,6 +195,10 @@ public function getInstallPathRelative(): string {
       return $this->data['install_path'];
     }
 
+    if ($this->getPackageName() == 'acquia/drupal-recommended-project') {
```

Comment:
Hack. As you can see below, I tried to instead key this off of a custom "project-template" Composer project type, but that didn't work. Maybe you can't make up project types willy-nilly.

The fixture creator already knows if the Sut is a project template via `isProjectTemplateSut()`, can we expose that so it's useful here? Or could that actually be a property of the package/sut somehow?